packages/create-ponder/tsup.config.ts

```diff
@@ -7,6 +7,7 @@ import { defineConfig } from "tsup";
 import { dependencies } from "./package.json";
 
 export default defineConfig({
+  name: "create-ponder",
```

Comment:
What does the name field do?

Reply:
Labels the information sent to the terminal